src/cogs/pixel_art/layer.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+import time
+
+import disnake
+from disnake.ext import commands
+from PIL import Image
+
+from main import tracked_templates
+from cogs.pxls_template.progress import autocomplete_templates
+from utils.arguments_parser import MyParser
+from utils.pxls.template_manager import get_template_from_url, parse_template, layer
+from utils.setup import stats
+from utils.discord_utils import image_to_file
+
+
+class Layer(commands.Cog):
+    def __init__(self, bot: commands.Bot) -> None:
+        self.bot: commands.Bot = bot
+
+    @commands.slash_command(name="layer")
+    async def _reduce(
+        self,
+        inter: disnake.AppCmdInter,
+        first_template: str = commands.Param(autocomplete=autocomplete_templates),
+        second_template: str = commands.Param(autocomplete=autocomplete_templates),
+        third_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
+        ),
+        fourth_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
+        ),
+        fifth_template: str = commands.Param(
+            default=None, autocomplete=autocomplete_templates
+        ),
+    ):
+        """Layer several images.
+
+        Parameters
+        ----------
+        first_template: A template link or name in the tracker.
+        second_template: A template link or name in the tracker.
+        ...etc
+        """
+        await inter.response.defer()
+        # Remove unused entries, equal to None
+        template_uris = [
+            temp
+            for temp in [
+                first_template,
+                second_template,
+                third_template,
+                fourth_template,
+                fifth_template,
+            ]
+            if temp
+        ]
+        templates = await self._clean(inter, template_uris)
+        await self._layer(inter, templates)
+
+    @commands.command(
+        name="layer",
+        description="Layer templates.",
+        usage="<first template name|link> <second template name|link> ...",
+        help="""
+            - `<template name|link>`: a template link or name in the tracker
```

Comment:
The option description should tell in which order the templates are layered (and same for the slash command description).
```suggestion
        usage="<templates>",
        help="""
            - `<templates>`: List of templates (URL or name) separated by a space (first goes above).
```